test/e2e/fixtures/build-default/src/index.ts

```diff
@@ -4,6 +4,7 @@ import './syntax/optional-chaining';
 import './syntax/jsx-import/JSX-import-JSX';
 
 import './syntax/async';
+export { merge, mergeAll } from 'lodash/fp';
 export { testGenerator } from './syntax/generator';
```

Comment:
```suggestion
export { testGenerator } from './syntax/generator';

export { kebabCase } from 'lodash';
export { merge, mergeAll } from 'lodash/fp';
```

The syntax imports are all grouped together intentionally. Please move the `lodash` imports below them. I'm also refactoring some of the syntax tests so this will ensure less merge conflicts

I added plain `lodash` here because we also want to add a regression test to ensure that hasn't broken with this change, since unfortunately that test didn't exist beforehand 😕